app/templates/base.html

```diff
@@ -95,6 +95,18 @@
       ga('create', 'UA-80458846-3', 'auto');
       ga('send', 'pageview');
 
+      /**
+      * Function that tracks a click on an outbound link in Analytics.
+      * This function takes a valid URL string as an argument, and uses that URL string
+      * as the event label. Setting the transport method to 'beacon' lets the hit be sent
+      * using 'navigator.sendBeacon' in browser that support it.
+      */
+      var trackOutboundLink = function(url) {
+         ga('send', 'event', 'outbound', 'click', url, {
+           'transport': 'beacon',
+           'hitCallback': function(){document.location = url;}
+         });
+      }
```

Comment:
Can we move this up into the header from body, as recommended in google analytics article?

> You should add this script in your page header, but not within the basic Analytics tracking code snippet.